scss/components/identifier.scss

```diff
@@ -101,19 +102,58 @@ $block: #{$fd-namespace}-identifier;
       }
     }
     //borders
-    &--circle {
+    &.#{$block}--circle {
       border-radius: 50%;
     }
     //thumbnail
-    &--thumbnail {
+    &.#{$block}--thumbnail {
       text-indent: -200%;
       overflow: hidden;
     }
     //background
-    &--transparent {
-      --fd-identifier-color: var(--fd-color-text-1);
+    &.#{$block}--transparent {
       --fd-identifier-background-color: transparent;
-      @include fd-var-color("color", $fd-color);
+      @include fd-var-color("color", map-get($fd-colors-text, 1), --fd-color-text-1);
       @include fd-var-color("background-color", transparent);
     }
 }
+
+.fd-has-background-color-accent-1 {
```

Comment:
These classes seem to be independent of identifier so they should either be renamed as part of identifier or be moved to a separate location to be shared.

Also, the `$fd-colors-accent` map has 15 colors defined -- we should include classes for each of them.